qpmodel/ExprSubquery.cs

```diff
@@ -36,6 +36,7 @@
 using qpmodel.logic;
 using qpmodel.physic;
 using qpmodel.utils;
+using IronPython.Compiler.Ast;
```

Comment:
Is this necessary? I can't find where it is used

Reply:
interesting, not sure when it get sneaked in - thanks for spotting it